device/dhcp/servers/v4/dhcpv4serverLeases.yaml

```diff
@@ -0,0 +1,76 @@
+components:
+  schemas:
+    DhcpServerV4.Leases:
+      description: >-
+        Configuration for DHCPv4 Leases.
+      type: object
+      x-status:
+        status: under_review
+      properties:
+        lease_time: 
+          description: >-
+            The Life Time length in seconds that is assigned to a lease if the requesting DHCP client does not specify a specific expiration time.
```

Comment:
As per my understanding, the current DHCPv4 model doesn't have a way to specify expiration time. 
So is it ok to mention "if the requesting DHCP client does not specify a specific expiration time." in the description ?